demo/ContainerInteractor.swift

```diff
@@ -133,6 +139,21 @@ extension ContainerInteractor: ContainerDelegate {
 
 extension ContainerInteractor: MenuDelegate {
     
+    private func modelPerformanceConfig(_ screen: Screen) -> ModelPerformanceConfig {
+        switch screen {
+        case .signsDetection, .objectDetection:
+            return .separate(segmentationPerformance: ModelPerformance(mode: .fixed, rate: .low),
```

Comment:
It may be interesting to show bicycles in the demo so we need to use merged model here too.